### PR TITLE
Minor fixes and add missing news entries

### PR DESCRIPTION
**Pull request by MariaSemple**
### Intent

> Describe briefly what problem this pull request resolves, or what new feature it introduces. Include screenshots of any new or altered UI. Link to any Github issues that are related. 

### Approach

> Describe the approach taken and the tradeoffs involved if non-obvious; add an overview of the solution if it's complicated.

### Automated Tests

> Indicate whether this change includes unit tests or integration tests, or link a work item or pull request to add those tests to another repo. If the change cannot or will not be covered by a test, indicate why.

### QA Notes

> Add additional information for QA on how to validate the change, paying special attention to the level of risk, adjacent areas that could be affected by the change, and any important contextual information not present in the linked issues. 

### Checklist

- [ ] If this PR adds a new feature, or fixes a bug in a previously released version, it includes an entry in `NEWS.md` 
- [ ] If this PR adds or changes UI, the updated UI meets [accessibility standards](https://github.com/rstudio/rstudio/wiki/Accessibility)
- [ ] A reviewer is assigned to this PR (if unsure who to assign, check Area Owners list)
- [ ] This PR passes all local unit tests

### CODE REVIEW

File: version/news/NEWS-2024.04.0-rel-chocolate-cosmos.md [hunk 1]
```diff
@@ -34,9 +34,9 @@
 - RStudio no longer includes spurious chunk markers from Copilot completion results (#13686)
 - Added new commands which toggle the maximized state of Console, Source, and Tab Group quadrants [Accessibility] (#7022)
 - RStudio now prints a warning if an overly-long line of console input is truncated when sending to R (#14420)
-- Updated zlib to 1.3.1 (rstudio/rstudio-pro#5971)
+- Updated zlib to 1.3.1 (rstudio-pro#5971)
```
Comment (MariaSemple) [inline version/news/NEWS-2024.04.0-rel-chocolate-cosmos.md:37]:
```suggestion
```

File: version/news/NEWS-2024.04.0-rel-chocolate-cosmos.md [hunk 2]
```diff
@@ -132,12 +134,16 @@
...
 ### Dependencies
 - Updated Ace to version 1.32.5 (#14227; Desktop + Server)
 - Updated Electron to version 28.2.6 (#14055; Desktop)
 - Updated GWT to version 2.10.0 (#11505; Desktop + Server)
 - Updated NSIS to version 3.09 (#14123; Windows Desktop)
 - Updated OpenSSL to version 3.1.4 (Windows Desktop)
-- Updated node.js to version 18.19.1 (#14373)
-
+- Updated nginx to 1.25.4 (rstudio-pro#5970)
+- Updated node.js to version 18.19.1 (#14373, rstudio-pro#5935)
```
Comment (MariaSemple) [inline version/news/NEWS-2024.04.0-rel-chocolate-cosmos.md:149]:
```suggestion
- Updated node.js to version 18.19.1 (#14373, rstudio-pro#5935)
- Updated zlib to 1.3.1 (rstudio-pro#5971)
```

File: version/news/NEWS-2024.04.0-rel-chocolate-cosmos.md [hunk 3]
```diff
@@ -34,9 +34,9 @@
 - RStudio no longer includes spurious chunk markers from Copilot completion results (#13686)
 - Added new commands which toggle the maximized state of Console, Source, and Tab Group quadrants [Accessibility] (#7022)
 - RStudio now prints a warning if an overly-long line of console input is truncated when sending to R (#14420)
-- Updated zlib to 1.3.1 (rstudio/rstudio-pro#5971)
+- Updated zlib to 1.3.1 (rstudio-pro#5971)
 - Vim: Fixed an issue where bracket highlight followed the bracket behind the cursor when in normal mode (#4152)
-- Updated nginx to 1.25.4 (rstudio/rstudio-pro#5970)
+- Updated nginx to 1.25.4 (rstudio-pro#5970)
```
Comment (MariaSemple) [inline version/news/NEWS-2024.04.0-rel-chocolate-cosmos.md:39]:
```suggestion
```

File: version/news/NEWS-2024.04.0-rel-chocolate-cosmos.md [hunk 4]
```diff
@@ -52,22 +50,24 @@
...
-- Prometheus metrics can now be enabled or disabled with `metrics-enabled=[1|0]` in `rserver.conf` (rstudio/rstudio-pro#5792)
-- Prometheus metrics can now be configured. For more details see [Promethes Metrics in the Workbench Admin Guide](https://docs.posit.co/ide/server-pro/2024.04.0/auditing_and_monitoring/prometheus_metrics.html) (rstudio/rstudio-pro#5791)
-- Add support for automatically provisioning users when using Okta or Microsoft Entra ID as an IdP. When using this feature, users can be assigned to Workbench from the IdP and do not need to be created as POSIX users. This feature also supports migrating from local, or ldap/sssd, users to IdP provisioned users. For more details see the [User Provisioning section of the Admin Guide](https://docs.posit.co/ide/server-pro/2024.04.0/user_provisioning/user_provisioning.html) (rstudio/rstudio-pro#2784)
-- Move the force-quit session option into the session info dialog (rstudio/rstudio-pro#3385)
+- Implemented Posit Product Documentation theme in Posit Workbench Administration Guide, Posit Workbench Licenses, Posit Workbench User Guide, and RStudio Desktop Administration Guide via Quarto extension (rstudio-pro/issues/5729)
```
Comment (GCRev) [inline version/news/NEWS-2024.04.0-rel-chocolate-cosmos.md:53]: For consistency:
```suggestion
- Implemented Posit Product Documentation theme in Posit Workbench Administration Guide, Posit Workbench Licenses, Posit Workbench User Guide, and RStudio Desktop Administration Guide via Quarto extension (rstudio-pro#5729)
```

**Review by GCRev — APPROVED** [review]
LGTM

**Review by jeffvroom — APPROVED** [review]
LGTM!

File: version/news/NEWS-2024.04.0-rel-chocolate-cosmos.md [hunk 5]
```diff
@@ -52,22 +50,24 @@
...
+- Prometheus metrics can now be enabled or disabled with `metrics-enabled=[1|0]` in `rserver.conf` (rstudio-pro#5792)
+- Prometheus metrics can now be configured. For more details see [Promethes Metrics in the Workbench Admin Guide](https://docs.posit.co/ide/server-pro/2024.04.0/auditing_and_monitoring/prometheus_metrics.html) (rstudio-pro#5791)
+- Add support for automatically provisioning users when using Okta or Microsoft Entra ID as an IdP. When using this feature, users can be assigned to Workbench from the IdP and do not need to be created as POSIX users. This feature also supports migrating from local, or ldap/sssd, users to IdP provisioned users. For more details see the [User Provisioning section of the Admin Guide](https://docs.posit.co/ide/server-pro/2024.04.0/user_provisioning/user_provisioning.html) (rstudio-pro#2784)
+- Move the force-quit session option into the session info dialog (rstudio-pro#3385)
+- Add support for setting Posit Workbench envrionment variables at the service level in an EnvironmentFile: `/etc/default/rstudio-server` (rstudio-pro#5337)
+- Add support for Slurm with Singularity and Apptainer (rstudio-pro#5236)
+-
```
Comment (melissa-barca) [inline version/news/NEWS-2024.04.0-rel-chocolate-cosmos.md:70]:
```suggestion
- Updated the Workbench JupyterLab 3 extension to v1.1.738 to upgrade NPM dependencies (rstudio-pro#5900)
```

**Review by melissa-barca — APPROVED** [review]
I added one missing entry. LGTM!

File: version/news/NEWS-2024.04.0-rel-chocolate-cosmos.md [hunk 6]
```diff
@@ -132,12 +132,17 @@
 - The Databricks pane now respects the `databricks-enabled=1` setting in the `rserver.conf` file even when running in legacy RStudio Server Pro mode (rstudio-pro#5695)
 - When launching sessions on Slurm, pending or failed sessions will no longer erroneously appear in the Workbench Jobs pane on the home page (rstudio-pro#4846)
 - VSCode sessions will now respect the `user-data-dir` option in `vscode.conf` when installing extensions in a user's directory on vscode session startup (rstudio-pro#6078)
+- Fixed an issue where `rstudio-server license-manager deactivate` would emit an error indicating incorrect usage of the `kill` command when `rstudio-server` is already stopped. (rstudio-pro#5549)
+- Fixed an error message in the Databricks pane of RStudio Pro sessions when Databricks authentication is not available (rstudio-pro#5583)
+- Fixed an issue where `rstudio-launcher` commands would not work when `RSTUDIO_CONFIG_DIR` or `XDG_CONFIG_DIRS` was set in systemctl overrides (rstudio-pro#4122)
+- Fixed an issue with the permissions on files uploaded from shared projects in RStudio Pro sessions (rstudio-pro#3134)
```
Comment (MariaSemple) [inline version/news/NEWS-2024.04.0-rel-chocolate-cosmos.md:138]:
```suggestion
- Fixed an issue with the permissions on files uploaded from shared projects in RStudio Pro sessions (rstudio-pro#3134)
- Fixed setting project sharing ACLs on parent directories for projects in subfolders (rstudio-pro#5309)
```

**Pull request merged**